templates/expo-template-default/package.json

```diff
@@ -17,29 +17,24 @@
     "@react-navigation/elements": "^2.6.3",
     "@react-navigation/native": "^7.1.8",
     "expo": "~54.0.0-preview.0",
-    "expo-constants": "~18.0.0",
     "expo-haptics": "~15.0.0",
     "expo-image": "~3.0.0",
-    "expo-linking": "~8.0.0",
     "expo-router": "~6.0.0-beta.0",
     "expo-splash-screen": "~31.0.0",
     "expo-status-bar": "~3.0.0",
     "expo-symbols": "~1.0.0",
-    "expo-system-ui": "~6.0.0",
     "expo-web-browser": "~15.0.0",
-    "react": "19.1.0",
-    "react-dom": "19.1.0",
+    "react": "~19.1.0",
+    "react-dom": "~19.1.0",
     "react-native": "0.81.0",
     "react-native-gesture-handler": "~2.28.0",
     "react-native-worklets": "~0.4.1",
     "react-native-reanimated": "~4.0.2",
-    "react-native-safe-area-context": "5.5.2",
+    "react-native-safe-area-context": "~5.6.0",
     "react-native-screens": "~4.14.0",
-    "react-native-web": "~0.21.0",
-    "react-native-webview": "13.15.0"
```

Comment:
this is intentional for people to try out dom components without necessary to install react-native-webview.

Reply:
<img width="1206" height="2622" alt="Simulator Screenshot - iPhone 16 Pro - 2025-08-15 at 18 29 35" src="https://github.com/user-attachments/assets/63928b03-a952-4999-9259-f38ea7921016" />

I think this checks out and is intuitive